### ISSUE TITLE
Stylization train error...

### ISSUE THREAD

**Issue by MixMe** [DmitryUlyanov/texture_nets#13]
Command:
th stylization_train.lua -style_image data/textures/style.jpg -train_hdf5 data/256.hdf5 -noise_depth 3 -model_name pyramid2 -normalize_gradients true -train_images_path data/256.hdf5 -content_weight 0.8 -backend cudnn

Error:
[libprotobuf WARNING google/protobuf/io/coded_stream.cc:537] Reading dangerously large protocol message.  If the message turns out to be larger than 1073741824 bytes, parsing will be halted for security reasons.  To increase the limit (or to disable these warnings), see CodedInputStream::SetTotalBytesLimit() in google/protobuf/io/coded_stream.h.
[libprotobuf WARNING google/protobuf/io/coded_stream.cc:78] The total number of bytes read was 574671192
Successfully loaded data/pretrained/VGG_ILSVRC_19_layers.caffemodel
conv1_1: 64 3 3 3
conv1_2: 64 64 3 3
conv2_1: 128 64 3 3
conv2_2: 128 128 3 3
conv3_1: 256 128 3 3
conv3_2: 256 256 3 3
conv3_3: 256 256 3 3
conv3_4: 256 256 3 3
conv4_1: 512 256 3 3
conv4_2: 512 512 3 3
conv4_3: 512 512 3 3
conv4_4: 512 512 3 3
conv5_1: 512 512 3 3
conv5_2: 512 512 3 3
conv5_3: 512 512 3 3
conv5_4: 512 512 3 3
fc6: 1 1 25088 4096
fc7: 1 1 4096 4096
fc8: 1 1 4096 1000
Setting up texture layer  	2	:	relu1_1	
Setting up texture layer  	7	:	relu2_1	
Setting up texture layer  	12	:	relu3_1	
Setting up texture layer  	21	:	relu4_1	
Setting up content layer	23	:	relu4_2	
Setting up texture layer  	30	:	relu5_1	
WARNING: Skipping content loss	
        Optimize        	
/home/ai2/torch/install/bin/luajit: stylization_train.lua:110: attempt to concatenate a nil value
stack traceback:
	stylization_train.lua:110: in function 'get_input_train'
	stylization_train.lua:141: in function 'opfunc'
	/home/ai2/torch/install/share/lua/5.1/optim/adam.lua:33: in function 'optim_method'
	stylization_train.lua:181: in main chunk
	[C]: in function 'dofile'
	.../ai2/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:145: in main chunk
	[C]: at 0x00405d50

Any idea ???

**Comment by DmitryUlyanov** [comment]
make sure hdf5 is prepared
